src/main/java/frc/robot/util/Node.java

```diff
@@ -54,48 +54,38 @@ public Node(Vision vision, Alliance alliance, int row, int column) {
           : NodeType.CONE;
 
     // Starting locations
-    double x = (alliance == Alliance.Blue ? 
-      FieldConstants.kBlueAllianceNodeStartX : 
-      FieldConstants.kRedAllianceNodeStartX
-    );
+    double x = 0;
+    if (alliance == Alliance.Blue) {
+      x = VisionConstants.kAprilTags.get(5).pose.getX() + FieldConstants.kAprilTagOffset + FieldConstants.kRobotDistanceFromNodeTape;
```

Comment:
Robot width is 26 inches, bumpers are 3 inches thick on each side, and there is about a 0.25 inch gap between the bumper and the chassis, so the robot width with bumpers on should be 32.5 inches total, not 33.5 inches.

Reply:
`kRobotWidthWithBumpers = Units.inchesToMeters(26 + 3.75 * 2);` should already be 33.5 inches